packages/at_onboarding_flutter/lib/screen/at_onboarding_generate_screen.dart

```diff
@@ -494,15 +495,19 @@ class _AtOnboardingGenerateScreenState
         if (!mounted) return;
         Navigator.pop(
             context, AtOnboardingResult.success(atsign: verifiedAtSign));
-      } else if (authResponse == AtOnboardingResponseStatus.serverNotReached) {
+      } else {
+        _onboardingService.setAtsign = previousAtsign;
+      }
+
+      if (authResponse == AtOnboardingResponseStatus.serverNotReached) {
```

Comment:
Same as revert above, slightly different semantic structure in this part of the code, so the change looks different